core/src/test/scala/org/apache/spark/deploy/ExternalShuffleServiceSuite.scala

```diff
@@ -14,14 +14,12 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-
-package org.apache.spark
+package org.apache.spark.deploy
```

Comment:
Move to `org.apache.spark.deploy` for better access to class within deploy package, e.g., `ApplicationRemoveTest`. And I think it should ideally be put in deploy package given that `ExternalShuffleService` itself also exists in deploy package.